code/datums/diseases/advance/symptoms/stinky.dm

```diff
@@ -0,0 +1,42 @@
+/*
+//////////////////////////////////////
+Stinky Syndicate
+	Somewhat Noticable.
+	Increases resistance.
+	Increases stage speed.
+	Very transmittible.
+	High Level.
+Bonus
+	Forces a spread type of AIRBORNE
+	with extra range!
+//////////////////////////////////////
+*/
+
+/datum/symptom/stinky
```

Comment:
What's the point of this symptom in general?

Reply:
Giving traitor virologist a huge buff if they spend tc.